sdk/sdk-api/src/main/java/mil/emp3/api/Circle.java

```diff
@@ -1,18 +1,17 @@
 package mil.emp3.api;
 
-import mil.emp3.api.abstracts.Feature;
-
 import org.cmapi.primitives.GeoCircle;
 import org.cmapi.primitives.GeoPosition;
 import org.cmapi.primitives.IGeoCircle;
 import org.cmapi.primitives.IGeoPosition;
 
 import java.util.List;
 
+import mil.emp3.api.abstracts.Feature;
 import mil.emp3.api.enums.FeatureTypeEnum;
 import mil.emp3.api.interfaces.IEmpBoundingBox;
 import mil.emp3.api.utils.EmpBoundingBox;
-import mil.emp3.api.utils.GeographicLib;
+import mil.emp3.api.utils.GeoLibrary;
```

Comment:
GeoLibrary should be deprecated, you will need to use GeographicLib for this class.